perfkitbenchmarker/packages/fio.py

```diff
@@ -115,12 +115,13 @@ def FioParametersToJob(fio_parameters):
                                 JOB_STONEWALL_PARAMETER)
 
 
-def ParseResults(job_file, fio_json_result):
+def ParseResults(job_file, fio_json_result, base_metadata={}):
```

Comment:
[Don't use mutable objects for argument defaults](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments).